Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
 
-gem 'json'
 gem 'acme-client'
+gem 'json'
```

Comment:
This change probably isn't necessary.

Reply:
That was probably rubocop rearranging names..